rbac/server/api/users.py

```diff
@@ -259,6 +259,23 @@
     )
 
 
+@USERS_BP.patch("api/users/<user_id>/roles/expired")
+@authorized()
+async def update_expired_roles(request, user_id):
+    """Manually expire user role membership"""
```

Comment:
Does this address the aspect of updating the blockchain?  I am concerned that it doesn't.

Reply:
Manual expiration is used to simulate membership expiration for demoing. See PR #957 for the blockchain aspect.